docs/src/about/FAQ.md

```diff
@@ -32,16 +35,29 @@ You can use test tokens in the same way you would use value-bearing tokens.
 
 For example with [PolkadotJS Apps](https://polkadot.js.org/apps/) you can create a transaction on behalf of your account, generate a signature with Signer and submit it to the network. All of this without keys ever leaving your offline device.
 
-### How do I keep my keys secure?
+## Networks
 
-Signer is a safe way to use your keys. However, that alone won't be enough to keep your keys secure. Devices break and get lost. This is why we always recommend to backup your seed phrases and derivation paths on paper. We are such big fans of paper backups that we even support a special tool to power your paper backup game by splitting your backups into shards called [Banana Split](https://bs.parity.io/).
+### How can I update metadata version for a network?
 
-### How do I know I am not interacting with malicious apps or actors?
+Please go to [Metadata Update Portal](https://metadata.parity.io/) and select network you need to update. Scan multipart QR code the same way you scan a transaction QR code. 
 
-Signer does not interact with network. The app itself does not have a way to check if an app or an account you interacting with is malicious. 
-If you use Signer with PolkadotJS Browser Extension, PolkadotJS Apps or Signer Component Browser Extension they will rely on community driven curated list of potentially less-than-honest operators: [https://polkadot.js.org/phishing/#](https://polkadot.js.org/phishing/#) to prevent you from interacting with certain sites and addresses. However, there are no limitations on use of Signer with other tools.
+Why do I need to update network versions at all?
+
+It's a safety feature. Substrate-based blockchain networks have capacity to be updated and otherwise changed; without recent metadata version of a network Signer won't be able to parse a transaction correctly, and you won't be able to read it and verify what you sign. Given that Signer is an app for an air-gapped device, you have to update the network version by using camera.
+
+### What networks does Signer support?
+
+From-the-shelf Party Signer supports Polkadot, Kusama and Westend networks. But it's not limited to these networks. More experienced users can generate metadata for any network to expand capability of Parity Signer.
+
+### How can I add a new network to Signer?
+
+Follow the [Add New Network](../tutorials/Add-New-Network.md) step-by-step guide.
```

Comment:
This is alternative to going to metadata portal - should be used if the network is not there or user doesn't like the portal